instrumentation/grpc/CHANGELOG.md

```diff
@@ -1,24 +1 @@
 # Release History: opentelemetry-instrumentation-grpc
-
-### v0.2.0 / 2025-01-16
-
-* BREAKING CHANGE: Set minimum supported version to Ruby 3.1
-
-* ADDED: Set minimum supported version to Ruby 3.1
-
-### v0.1.3 / 2024-09-11
-
-* FIXED: Fix error in handling of non-gRPC errors
-* FIXED: Fix error in method signature for OpenTelemetry::Instrumentation::Grpc.client_interceptor [#1](https://github.com/hibachrach/opentelemetry-instrumentation-grpc/pull/1)
-
-### v0.1.2 / 2024-06-26
-
-* FIXED: Align span naming with spec
-
-### v0.1.1 / 2024-06-26
-
-* FIXED: Fix `uninitialized constant Interceptors` error
-
-### v0.1.0 / 2024-06-18
-
-* Initial release
```

Comment:
@michal-kazmierczak - Could you add the CHANGELOG information back from version 0.1.3 and below? We're now set up to release the next version of the gem in step with the existing versioning and would like to maintain the README.